src/main/g8/README.md

```diff
@@ -14,15 +13,26 @@ The intention is to use `sbt new` on this library's exterior g8 template, then t
 - `sbt-native-packager` plugin for **building a linked binary** _(builds ahead-of-time using GraalVM statically linking and building all dependencies in a docker image via `sbt graalvm-native-image:packageBin`)_
 - `scalatest` for **testing** _(default test runner via `sbt test`)_
 - `sbt-scoverage` for **test coverage** _(statement-level coverage via `sbt coverage`)_
-- `scalalogging -> slf4j -> log4j2` for **logging** _(scala code -> interface -> mechanism)_
+- `scalalogging -> logback-classic` for **logging** _(`logback-classic` is used as a backend, but it is very easy to [replace this with something like log4j2](https://github.com/sbchapin/quickstart-scala-sbt.g8/blob/master/src/main/g8/build.sbt#L19))_
 - `scopt` for command-line **argument parsing**
 - `sbt-updates` for **keeping up-to-date** _(dependency updates via `sbt dependencyUpdates`)_
 - `sbt-dependency-graph` for **understanding your package** _(dependency graph via `sbt dependencyBrowseGraph` and dependency stats via `sbt dependencyStats`)_
 
-Check out the [Libs](#markdown-header-libs) below for more details on each.
+Check out the [Libs](#libs) below for more details on each.
```

Comment:
Damn, he be testin!  Thanks for the thoroughness of this PR. 😀

Reply:
🙇